rego/rego.go

```diff
@@ -1107,6 +1108,13 @@ func EnablePrintStatements(yes bool) func(r *Rego) {
 	}
 }
 
+// Add Strict Validation
```

Comment:
Some linters complain about this style of comment, let's go with
```suggestion
// Strict enables or disables strict-mode in the compiler
```
or something like that

Reply:
Done!